docs/support/account.md

```diff
@@ -7,6 +7,17 @@ If your TechPass login ID's domain is ```techpass.gov.sg```, your TechPass accou
 
 </details>
 
+<details>
+<summary>How do I terminate my own account, or help my teammate terminate his/her account?</summary>
+
+a. Create a [service request](https://go.gov.sg/seed-techpass-support).
+
+?> In the service request form, select **Service Request** as **Ticket Type** and select **User has left the organisation, I would like to terminate his account** as **Service Requests**. 
+
+b. Fill in the email associated with the TechPass account to be terminated in the **Email id for termination** field.
```

Comment:
We had a discussion about changing the wording, please change the "Email id for termination" to "Email address for termination"

Reply:
done